src/utils.ts

```diff
@@ -1,9 +1,19 @@
 import { app, BrowserWindow, dialog } from 'electron'
+import config, { ConfigKey } from './config'
 
 export function getMainWindow(): BrowserWindow {
   return BrowserWindow.getAllWindows()[0]
 }
 
+export function setAppMenuBarVisibility(): void {
+  const mainWindow = getMainWindow()
+  if (mainWindow) {
```

Comment:
When would this conditional be false?

Reply:
I cannot think of any, my bad. It's a redundant check and will be removed, thanks for pointing it out.